docs/source/torch/auto_deploy/advanced/logging.md

```diff
@@ -0,0 +1,14 @@
+# Logging Level
+
+Use the following env variable to specify the logging level of our built-in logger ordered by
```

Comment:
Suggestion: “Use the following env variable to specify the logging level of the built-in logger, ordered by decreasing verbosity:”